config/runtime/fstests.jinja2

```diff
@@ -1,32 +1,58 @@
 {# -*- mode: Python -*- -#}
 {# SPDX-License-Identifier: LGPL-2.1-or-later -#}
- 
+
 {%- extends 'base/python.jinja2' %}
- 
+
 {%- block python_imports %}
 {{ super() }}
 import subprocess
 {%- endblock %}
- 
+
 {%- block python_globals %}
 {{ super() }}
 REVISION = {{ revision }}
 {% endblock %}
- 
-{%- block python_body %}
-def check_kvm_xfstest_env():
-   try:
-       result = subprocess.run("kvm-xfstests --help", check=True, shell=True)
-       if result.returncode == 0:
-           print(f"KVM-xftests found.")
-           return True
-       else:
-           return False
-   except Exception as e:
-       print(f"Raised exception: {e}")
-       return False
- 
-def main(argv):
-   print("Checking if KVM-xfstests is available...")
-   check_kvm_xfstest_env()
+
+{% block python_job_constr -%}
+REVISION, {{ super() }}
+{%- endblock %}
```

Comment:
I think this can be dropped, the revision is not used in the `Job` constructor.  It's needed for the `kver` test plan.